docker-compose.yml

```diff
@@ -1,31 +1,59 @@
 zk:
-  image: bobrik/zookeeper
+  image: jplock/zookeeper:3.4.6
   ports:
    - 2181:2181
-  environment:
-    ZK_CONFIG: tickTime=2000,initLimit=10,syncLimit=5,maxClientCnxns=128,forceSync=no,clientPort=2181
-    ZK_ID: 1
 
 master:
   image: mesosphere/mesos-master:0.23.0-1.0.ubuntu1404
   net: host
   environment:
-    MESOS_ZK: zk://127.0.0.1:2181/mesos
-    MESOS_HOSTNAME: 192.168.59.103
-    MESOS_IP: 192.168.59.103
+    MESOS_ZK: zk://192.168.99.100:2181/mesos
+    MESOS_HOSTNAME: 192.168.99.100
+    MESOS_IP: 192.168.99.100
     MESOS_QUORUM: 1
     MESOS_CLUSTER: docker-compose
     MESOS_WORK_DIR: /var/lib/mesos
 
-slave:
+slave1:
+  image: mesosphere/mesos-slave:0.23.0-1.0.ubuntu1404
+  net: host
+  pid: host
+  environment:
+    MESOS_MASTER: zk://192.168.99.100:2181/mesos
+    MESOS_HOSTNAME: 192.168.99.100
+    MESOS_IP: 192.168.99.100
+    MESOS_CONTAINERIZERS: docker,mesos
+    MESOS_PORT: 5051
+  volumes:
+    - /sys/fs/cgroup:/sys/fs/cgroup
+    - /usr/local/bin/docker:/usr/bin/docker
+    - /var/run/docker.sock:/var/run/docker.sock
+
+slave2:
```

Comment:
What's the reason to have 2 slaves that share the same resources, including ip address?